### ISSUE TITLE
Documentaion

### ISSUE THREAD

**Issue by cbmainz** [tothi/php-mailinglist#1]
Is there any documentation on how to implement your code?

**Comment by tothi** [comment]
no, there isn't. ;) it was made primarily for private use. but i can upload some examples, if it helps.

**Comment by cbmainz** [comment]
I understand. But examples would be nice. 

Thanks in advance 
Christopher

**Comment by tothi** [comment]
ok, added a tiny example (actually which i used for testing ;) )
